# Typescript type enforcement for Aura supporting sfdx-typegen (functions-should-include-typed-this)

## Rule Details

This rule aims to ensure controller and helper functions have "this" defined.

Examples of **incorrect** code for this rule:

```typescript
// fill me in
```

Examples of **correct** code for this rule:

```typescript
// fill me in
```

## Further Reading

If there are other links that describe the issue this rule addresses, please include them here in a bulleted list.
